readme.md

```diff
@@ -59,6 +59,8 @@ Default: `100`
 
 Truncate the filename to the given length.
```

Comment:
Here we refer to the "filename" as the whole thing including extension, but below you refer to it as only the part without the extension.